### ISSUE TITLE
Continuum mosaic: Edge/noise problems between fields

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#411]
![image](https://github.com/ACES-CMZ/reduction_ACES/assets/143715/b132aee9-9ad7-4a12-b0d2-5d3c150875b2)

(fields ar (center) and k (bottom-right))

This looks to be caused by significant depth differences. "I’m just looking at the two weblogs, and field k has a lower RMS, significantly smaller beam, and > 2x aggregate bandwidth compared to field ar" from Dan

**Comment by d-l-walker** [comment]
### Weblog summaries for continuum images

#### Field ar
![Screenshot 2024-01-24 at 14 58 47](https://github.com/ACES-CMZ/reduction_ACES/assets/13015266/7880f9c8-e4e8-44a7-af38-ee9ff6ba24fd)

#### Field k
![Screenshot 2024-01-24 at 14 58 56](https://github.com/ACES-CMZ/reduction_ACES/assets/13015266/8eecff58-bd6e-4add-9560-562d32cf0c49)

**Comment by keflavich** [comment]
Right, that's a huge difference in sensitivity: 83 uJy/beam in 1.57x0.929 = 56 uJy/beam in a 2.02x1.55" beam, so about 2x better noise.

**Comment by keflavich** [comment]
This issue manifests in part as excess noise along the edges of peak intensity maps.

I had started working on debugging this at some point, but didn't track my work well:

https://github.com/ACES-CMZ/reduction_ACES/blob/4ed2804f883ddf148d02fbd18eb97812c078f5dc/aces/imaging/debug_mosaic_weights.py

**Comment by keflavich** [comment]
e.g., this:

<img width="328" alt="Image" src="https://github.com/user-attachments/assets/5d5f57b2-dafc-42c7-bc68-7b23f8b9af29" />

**Comment by keflavich** [comment]
The problem was partly or entirely (not sure yet) that the weight images are a different shape from the feathered (12m+7m+TP) cubes, since the latter are cropped down to include only significant pixels.

I have possibly solved this with a combination of fixes to the ACES pipeline, [spectral-cube](https://github.com/radio-astro-tools/spectral-cube/pull/868), and [reproject](https://github.com/astropy/reproject/pull/474).

**Comment by keflavich** [comment]
This seems _mostly_ solved, but not perfectly.  Is it possible that the remaining problem is just that some fields are under-weighted?  Maybe we should normalize the peak weights?

https://github.com/ACES-CMZ/reduction_ACES/blob/28a2bb4c8be3d51dcab7ca90c6c1c51edac416ad/aces/imaging/debug_mosaic_weights.py is the tool to test this.



<img width="530" alt="Image" src="https://github.com/user-attachments/assets/dcc632d5-e070-4498-b279-42e1fb78f9af" />
